src/main/java/mediathek/tool/MVFunctionSys.java

```diff
@@ -75,13 +75,13 @@ public static String pathProgramIcons() {
     public static String pathSenderIcons() {
         return getPathJar() + Konstanten.VERZEICHNIS_SENDER_ICONS;
     }
-
+    @Deprecated
     public static String getProgVersionString() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1133)